frontend/package.json

```diff
@@ -30,9 +30,8 @@
         "@heroicons/react": "^1.0.6",
         "@monaco-editor/react": "^4.6",
         "@primer/octicons-react": "^17",
-        "@redpanda-data/ui": "^3.31",
+        "@redpanda-data/ui": "^3.37.0-alpha.0",
         "@textea/json-viewer": "^1.24.4",
-        "antd": "^4.21",
```

Comment:
Awesome!

Reply:
It might be worth checking what were the dependencies that we had to bring in to the codebase because of `antd`, and remove those as well?

Reply:
I can check that, do you have some specific things in mind or it's more of a general comment?

Reply:
I just thought we could analyse the bundle or at least check what are the main packages that antd depends on, and if they still need to be brought into our build